src/components/rating/Rating.tsx

```diff
@@ -66,16 +80,21 @@ const Rating = ({ isResponseGenerated, feedbackId }: RatingProps) => {
     return null;
   }
 
+  const openFeedbackModal = async (feedback: FeedbackPayload) => {
+    return setIsFeedbackOpen(true);
+  };
+
+  const sendDetailedFeedback = async (feedback: FeedbackPayload) => {
```

Comment:
I think we can merge this function with `sendFeedback` fn. Only diff is the feedback input which can be accounted for in the `sendFeedback` fn

Reply:
Where is the fn used?

Reply:
Definitely. Looking at the code, I'm not using the function anywhere so it will be removed. An oversight on my part